public/docs/ts/latest/guide/template-syntax.jade

```diff
@@ -738,22 +679,20 @@ code-example(format="", language="html").
   Instead of an element property between brackets, we start with the prefix `style`,
   followed by a dot (`.`) and the name of a CSS style property: `[style.style-property]`.
 
-// #enddocregion other-bindings-style-1
 +makeExample('template-syntax/ts/app/app.component.html', 'style-binding-1')(format=".")
-// #docregion other-bindings-style-2
 :marked
   Some style binding styles have unit extension. Here we conditionally set the font size in  “em” and “%” units .
-// #enddocregion other-bindings-style-2
 +makeExample('template-syntax/ts/app/app.component.html', 'style-binding-2')(format=".")
 
-// #docregion other-bindings-style-3
 .l-sub-section
   :marked
     While this is a fine way to set a single style,
     we generally prefer the [NgStyle directive](#ngStyle) when setting several inline styles at the same time.
-// #enddocregion other-bindings-style-3
 
-// #docregion event-binding-1
+.l-sub-section
+  :marked
+    Note that the _style property_ name can be written in camcel case, such as `fontSize`.
```

Comment:
Both are used elsewhere, about equally, in the docs. I'll use "camelCase". Let me know if you have objections.